android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -792,6 +794,13 @@ public void onResume() {
 
         PostTask.postTask(
                 TaskTraits.BEST_EFFORT_MAY_BLOCK, () -> { BraveStatsUtil.removeShareStatsFile(); });
+        int appOpenCountForWidgetPromo = SharedPreferencesManager.getInstance().readInt(
```

Comment:
@deeppandya app open count already there in finishNativeInitialization, if we want to consider onResume as openOpenCount, I hope we can move that to here and use same preference here.

Reply:
@sujitacharya2005 i didn't want to break existing feature's count so i added a new count which affects only widget